benchmark/src/main/java/org/eclipse/rdf4j/benchmark/FastRdfsForwardChainingSailBenchmark.java

```diff
@@ -0,0 +1,34 @@
+/*******************************************************************************
+ * Copyright (c) 2016 Eclipse RDF4J contributors.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Distribution License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *******************************************************************************/
+
+
+package org.eclipse.rdf4j.benchmark;
+
+import org.eclipse.rdf4j.repository.sail.SailRepository;
+import org.eclipse.rdf4j.sail.inferencer.fc.FastRdfsForwardChainingSail;
+import org.eclipse.rdf4j.sail.memory.MemoryStore;
+
+/**
+ * @author Håvard Mikkelsen Ottestad
+ */
+public class FastRdfsForwardChainingSailBenchmark extends RdfsBenchmark {
+
+
```

Comment:
If you import the settings from https://github.com/eclipse/rdf4j/tree/master/eclipse-settings into IntelliJ, they should mostly map okay, based on my limited IntelliJ experience. Any incompatibilities will be fairly small after that and we can reformat using Eclipse to fix them without a large diff.